ext/opentelemetry-ext-django/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+
+from django import setup
+
+from opentelemetry.ext.django import DjangoInstrumentor
+
+_django_instrumentor = DjangoInstrumentor()
+
+
+def pytest_sessionstart(session):  # pylint: disable=unused-argument
+    environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.settings")
```

Comment:
Any specific reason to do it here and not in `setUpClass` and `tearDownClass`?

Reply:
Yes, it would be too late. By the time these environment variables are set the OpenTelemetry configuration class and Django would be loaded already, missing them.